server/src/services/boards/sessions.go

```diff
@@ -289,6 +289,20 @@ func (s *BoardSessionService) UpdatedSession(board uuid.UUID, session database.B
 			logger.Get().Errorw("unable to broadcast updated board session", "err", err)
 		}
 	}
+
+	columns, err := s.database.GetColumns(board)
+	if err != nil {
+		logger.Get().Errorw("unable to get columns on a updatedsession", "err", err)
+	}
+
+	err = s.realtime.BroadcastToBoard(board, realtime.BoardEvent{
+		Type: realtime.BoardEventColumnsUpdated,
+		Data: dto.Columns(columns),
+	})
+	if err != nil {
+		logger.Get().Errorw("unable to broadcast update columns following a updatedsession", "err", err)
+	}
+
```

Comment:
Have you checked if the notes in the hidden column are displayed when a user is promoted to moderator?
As far as I know, BoardEventColumnsUpdated only sends information about the columns, but not about the notes in them

Reply:
Good catch! 🤝 

Seems that it was to late yesterday 😅 
Fixed it!